schemas/protocol/codebook/entities.ts

```diff
@@ -21,6 +23,8 @@ export const NodeIcons = [
   ...lucideIcons,
 ] as const;
```

Comment:
As in previous comment, this new functionality should replace the existing icons, so `NodeIcons` is just `lucideIcons`.